config/templates/kava/upgrade-v44/initstate/.kava/config/genesis.json

```diff
@@ -896,8 +896,27 @@
         ],
         "swap_fee": "0.001500000000000000"
       },
-      "pool_records": [],
-      "share_records": []
+      "pool_records": [
```

Comment:
We also have to ensure the swap module account has a balance equal to the total pool reserves - set here https://github.com/Kava-Labs/kvtool/blob/57165dc2214aeddde380725bdf9dabcaefff8fd1/config/templates/kava/upgrade-v44/initstate/.kava/config/genesis.json#L322

Reply:
Hmm after adding the balances and updating bank supply, I get a broken invariant: 

```
kavanode_1  | panic: invariant broken: swap: pool reserves broken invariant
kavanode_1  | pool reserves do not match module account
kavanode_1  |
kavanode_1  | 	CRITICAL please submit the following transaction:
kavanode_1  | 		 tx crisis invariant-broken swap pool-reserves
```

It runs okay without any coins set for the swap module account

Reply:
ah good catch, fixed at https://github.com/Kava-Labs/kava/pull/1064